### DISCUSSION TITLE
Build error from source version 21.2.1

### DISCUSSION THREAD

**Discussion by fanyify** [dbeaver/dbeaver#14026] [DBeaver Development] [answered]
#### System information: 
- Operating system (distribution) and version CentOS7 x86_64
- DBeaver version: 21.2.1
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
Build from source failed with a missing required bundle error `net.sf.opencsv`. Also checked with `https://dbeaver.io/eclipse-repo` this link, it does do not have this bundle!!

#### Steps to reproduce, if exist:
1. Download source from github and select tag as 21.2.1
2. run mvn clean package in dbeaver directory

#### Include any warning/errors/backtraces from the logs

[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.jkiss.dbeaver.ext.greenplum 1.0.66.qualifier
[ERROR]   Missing requirement: org.jkiss.dbeaver.ext.postgresql 2.1.138.qualifier requires 'osgi.bundle; net.sf.opencsv 0.0.0' but it could not be found
[ERROR]   Cannot satisfy dependency: org.jkiss.dbeaver.ext.greenplum 1.0.66.qualifier depends on: osgi.bundle; org.jkiss.dbeaver.ext.postgresql 0.0.0
[ERROR]

**Comment by ShadelessFox** [comment, marked as answer]
Hello @fanyify,

The mentioned bundle is indeed missing because we've excluded it and from now are shipping own copy of it.

It happened after the release of 21.2.1, so you will have to build from `devel` branch or wait for the next release.

**Comment by fanyify** [comment]
Ok, thanks for your reply.
